scripts/quests/windurst/THF_AF1_The_Tenshodo_Showdown.lua

```diff
@@ -1,7 +1,7 @@
 -----------------------------------
 -- The Tenshodo Showdown
 -----------------------------------
--- !addquest 2 69
+-- Log ID: 2, Quest ID: 69
```

Comment:
`!addquest 2 69` is here so you can quickly copy/paste it for testing, same as with the commands below it

Reply:
Gotcha - plopped it back in